common/test/acceptance/pages/lms/video/video.py

```diff
@@ -661,6 +663,11 @@ def state(self):
         state_selector = self.get_element_selector(CSS_CLASS_NAMES['video_container'])
         current_state = self.q(css=state_selector).attrs('class')[0]
 
+        # For troubleshooting purposes show what the current state is.
+        # The debug statements will only be displayed in the event of a failure.
+        logging.debug("Current state of '{}' element is '{}'".format(state_selector, current_state))
```

Comment:
Reviewer note: since this is a debug statement it will not get printed out except in the case of a test failure. Then it will show us the video state, which will help us to troubleshoot [TNL-454 video tests fail intermittently with Promise not satisfied: State is pause / play](https://openedx.atlassian.net/browse/TNL-454)

I'm trying to determine if the root cause of the flakiness is that the button was clicked too early (JS / click handlers not installed) or perhaps we just aren't waiting long enough (this might be the case, as screenshots sometimes still show a spinner).